docs/BEMAN_LIBRARY_MATURITY_MODEL.md

```diff
@@ -6,26 +6,29 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 `The Beman maturity model` helps developers quickly assess the production readiness of Beman libraries by classifying them based on development phase and interface stability.
 
-<img src="../images/beman_flow-beman_library_maturity_model.png">
+![](/images/beman_flow-beman_library_maturity_model.png)
 
 ### Under development and not yet ready for production use.
-<img src="../images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;"> These libraries may deviate from the Beman Standard due to incompleteness, lack of testing, inconsistencies with the specification, or other non-conformances.
+
```

Comment:
we're dropping all the images?

Reply:
ok I see images in the preview

Reply:
@JeffGarland , are we good here?

Reply:
Without this patch, we cannot automatically sync the docs. The git pages static build will fail